backend/frontend/views.py

```diff
@@ -274,6 +277,59 @@ def action_start_card(request, card_id):
     )
 
 
+def user_can_request_review(logged_in_user):
+    if len(helpers.agile_card_reviews_outstanding(logged_in_user)):
+        return False
+    if len(helpers.pull_request_reviews_outstanding(logged_in_user)):
+        return False
```

Comment:
I think we can ignore these checks if we want to remind the learners to review outstanding reviews before starting in the UI like we currently are. It's probably possible but I think the button should be there for us to have an opportunity to remind them.

I don't know if I'm explaining this well or not, so let me know if it doesn't make sense

Reply:
I did not quite get you, but this isn't enforced at the model level, and as per the current setup, one shouldn't be able to move it to IR, hence its presence here. 

Also, the fact that this is on the "action" view, not the main one in charge of displaying the cards, means there's still that visual cue for the outstanding IR cards.

Let me know if we're talking about the same thing 😅

Reply:
Lol, let me try again.

The color being a visual cue is all well and good. I just think it is better to be explicit about why your card is not able to move to review with something like a toast or a modal. Especially for new users who just know they can move cards from 1 column to the other.

Also, my thinking was around how easy it would be to implement that "explicit UI feedback" mentioned above. My thinking is we would still have to do some checks with the above helper functions again since the ones used in the above function only checks a lot of stuff including the other permissions.

Reply:
Got it